react/src/components/GeneSearch.tsx

```diff
@@ -40,6 +42,9 @@ interface GeneSearchProps {
     onChange: (geneName: string) => void;
 }
 
+/* Typeguard */
+const isMultipleGenes = (arg: Gene | Gene[]): arg is Gene[] => (arg as Gene[]).length >= 0;
```

Comment:
`Array.isArray` is probably a better check

Reply:
Good point!